test/functional/normalization/loglevel/journal_logs_test.go

```diff
@@ -0,0 +1,62 @@
+package loglevel
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+	"github.com/openshift/cluster-logging-operator/test/framework/functional"
+	testfw "github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+)
+
+var _ = Describe("[functional][normalization][loglevel] tests for message format of journal logs", func() {
+
+	var (
+		framework *functional.CollectorFunctionalFramework
+	)
+
+	BeforeEach(func() {
+		framework = functional.NewCollectorFunctionalFrameworkUsingCollector(testfw.LogCollectionType)
+		functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+			FromInput(logging.InputNameInfrastructure).
+			ToFluentForwardOutput().
+			FromInput(logging.InputNameAudit).
+			ToElasticSearchOutput()
```

Comment:
This was added as part of another test to ensure we constructed pipelines correctly.  It may not be required here but it does offer a way to ensure we get configs "correct" when multiplexing across multiple pipelines